README.md

```diff
@@ -15,7 +15,7 @@ web.listen-address              | Address to listen on for web interface and tel
 web.telemetry-path              | Path under which to expose metrics.
 exporter.aurora-url             | [URL](#aurora-url) to an Aurora scheduler or ZooKeeper ensemble.
 zk.path                         | The path for the aurora scheduler znode, this defaults to `/aurora/scheduler`
-exporter.bypass-leader-redirect | Don't follow redirects to the leader instance.
+exporter.bypass-leader-redirect | Don't follow redirects to the leader instance. Ignored for quotas scraping due technical reasons.
```

Comment:
Can you expand on this and explain that non-leading masters results in an internal server?